rolling-shutter/keyperimpl/shutterservice/newblock.go

```diff
@@ -56,6 +55,8 @@ func (kpr *Keyper) maybeTriggerDecryption(ctx context.Context, block *syncevent.
 		return errors.Wrap(err, "failed to query non decrypted identity registered events from db")
 	}
 
+	kpr.latestTriggeredTime = &block.Header.Time
+
```

Comment:
`kpr.latestTriggeredTime` should be updated after the db query is successfully executed. If we update it before the db query and for some reason there was an error while executing the query, we might miss some previous registered events since the `kpr.latestTriggeredTime` was already updated with new blocks timestamp.